Documentation/k8s-pre-reqs.md

```diff
@@ -154,6 +154,17 @@ imagePullSecrets:                # here are the new
 
 After doing this for all service accounts all pods should be able to pull the image from your registry.
 
+### Procedure for adding imagePullSecrets to service accounts if Helm Charts is used for Rook Installation :
+In [Values.yaml](https://github.com/rook/rook/blob/master/cluster/charts/rook-ceph/values.yaml) populate imagePullSecrets, it will passed to all service accounts.
```

Comment:
nit: add a blank line between the title and the paragraph. Also, remove extra lines before the next title.
```suggestion
In [Values.yaml](https://github.com/rook/rook/blob/master/cluster/charts/rook-ceph/values.yaml) populate `imagePullSecrets` to be set on all service accounts.
```